pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -75,85 +73,16 @@ func (rs RunSpec) GetParam(name string) *v1beta1.Param {
 	return nil
 }
 
-type RunStatus struct {
-	duckv1.Status `json:",inline"`
+const (
+	// RunReasonCancelled must be used in the Condition Reason to indicate that a Run was cancelled.
+	RunReasonCancelled = "RunCancelled"
+)
 
-	// RunStatusFields inlines the status fields.
-	RunStatusFields `json:",inline"`
-}
+// RunStatus defines the observed state of Run.
+type RunStatus = runv1alpha1.RunStatus
```

Comment:
Any special reason for moving only the status part to its own package?
I think it would make sense to have them all in the same package outside of pipeline?

Reply:
This was discussed earlier with @vdemeester:

> Moving the RunSpec is harder because it uses v1beta1.Params and v1alpha1.TaskRef. This means a lot of refactoring which must be done carefully to avoid breaking these widely used types. I discussed offline with Vincent and he agreed this could be considered for a follow on to this PR.

Personally I feel there is not much value in moving the rest of it.  There's bigger fish to fry.

Reply:
Ah, I missed that discussion, I agree.
I small request though, as reading the code without this context might have the same question. Could you file an issue about the follow-up refactor, and add a comment into the code to refer to it? (A follow up PR also works)